Dockerfile

```diff
@@ -68,6 +68,7 @@ ADD backend/scripts/restore.sh /usr/local/bin/restore
 ADD backend/scripts/import.sh /usr/local/bin/import
 RUN chmod +x /entrypoint.sh /usr/local/bin/backup /usr/local/bin/restore /usr/local/bin/import
 
+COPY --from=build-backend /build/backend/app/notify/templates /srv/templates
```

Comment:
Have you tried replacing the template for a production binary with something else? I don't know if that works (and if we should provide a remark binary with them). In other projects Umputun bundles such dependencies inside binary, let's see his opinion on that on final review.

Reply:
Yes, but he provide opportunity to load templates by path https://github.com/akellbl4/remark/blob/7b67b10d5ddcfad80c58a4b76d6b065458320371/backend/app/cmd/server.go#L690

Also, he is not able to put all frontend to binary :) Because I think it's really comfortable to edit templates in separated files.

Reply:
It's good if you checked that it's really possible to change template and get a new look (and an error if it's incorrect).

Reply:
I tried to connect email_templates from disk like volume and it is works as well